core/src/main/java/org/mobilitydata/gtfsvalidator/notice/ValidationNotice.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.notice;
+
+import java.util.Map;
+
+/**
+ * ValidationNotice is the base class for all validation errors and warnings
+ * related to the content of a GTFS feed.
+ * <p>
+ * This is the parent class for the most of notices, such as
+ * {@link DuplicatedColumnNotice}, {@link NumberOutOfRangeError} or notices
+ * produced by user-implemented validators.
```

Comment:
I'm not sure "user-implemented" is clear here - maybe "3rd party" instead, if that is what is intended (i.e., validators not contained in the MobilityData repo)?

Reply:
Good point. Rephrased.